cloudvolume/storage/storage_interfaces.py

```diff
@@ -191,7 +191,9 @@ def put_file(self, file_path, content, content_type, compress, cache_control=Non
     blob = self._bucket.blob( key )
 
     # gcloud disable brotli until content-encoding works
-    if compress:
+    if compress == "br":
+      raise TypeError("Brotli unsupported on google cloud storage")
```

Comment:
Sorry to be overly pedantic, but I think this is probably more of a `ValueError` than a `TypeError`. However, it would be better to create a new exception class, something like `exceptions.UnsupportedCompressionType` in `exceptions.py` which is more self-explanatory in code that tests for it in a try-catch and also more specific.

Reply:
makes sense, was in a meeting when i coded that up. fixed